packages/docs-page/server/index.ts

```diff
@@ -68,10 +65,8 @@ export function getStaticGenerationFunctions(
           revalidate: opts.revalidate,
         }
       } catch (err) {
-        console.error(err)
-        return {
-          notFound: true,
```

Comment:
I think we'd want `notFound: true` if the API returns a 404, right?

Reply:
Hm, yes...

Reply:
Personal notes:

https://waypoint-git-kevin-versioned-docs-hashicorp.vercel.app/docs/foobar (expected non-existent page)
![image](https://user-images.githubusercontent.com/26389321/140988414-f781080b-1d21-4aa5-8cfa-b0e3ace6be37.png)

Reply:
Then after catching `404` and returning `{ notFound: true }`
https://waypoint-git-kevin-versioned-docs-hashicorp.vercel.app/docs/foobar
![image](https://user-images.githubusercontent.com/26389321/140988683-760f4e43-47f8-4c15-bb63-d67b37172d40.png)